scripts/auto-rebase/crypto_scan.sh

```
@@ -0,0 +1,83 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+SCANNER_IMAGE="images.paas.redhat.com/exd-sp-guild-security/rh-crypto-scanner-image:latest"
+REPOROOT="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/../..")"
+CBOM_OUTPUT="${REPOROOT}/cbom-microshift.json"
+SPDX_OUTPUT="${REPOROOT}/sbom-microshift-crypto.spdx.json"
+
+IMAGE_DIR=$(mktemp -d)
+SCANNER_DIR=$(mktemp -d)
+SCAN_DIR=$(mktemp -d)
+trap "rm -rf ${IMAGE_DIR} ${SCANNER_DIR} ${SCAN_DIR}" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)